tsconfig.json

```diff
@@ -1,25 +1,17 @@
 {
-  "exclude": [
-    "example",
-    "node_modules",
-    "glyphmaps",
-    "scripts",
-  ],
+  "exclude": ["example/**"],
   "compilerOptions": {
     "baseUrl": ".",
     "paths": {
-      "@draftbit/ui": [
-        "packages/react-native-jigsaw/src/index"
-      ]
+      "@draftbit/*": ["packages/*/src"]
```

Comment:
Typescript will resolve these packages against the monorepo.  We overwrite this in each module with a `tsconfig.build.json` file so that builds for publication have real package references.